src/diagrams/class/classDb.js

```diff
@@ -43,13 +43,39 @@ export const addRelation = function(relation) {
   relations.push(relation);
 };
 
+/**
+ * Adds an annotation to the specified class
+ * Annotations mark special properties of the given type (like 'interface' or 'service')
+ * @param className The class name
+ * @param annotation The name of the annotation without any brackets
+ * @public
+ */
+export const addAnnotation = function(className, annotation) {
+  classes[className].annotations.push(annotation);
```

Comment:
I'm unhappy about that. This will throw an exception if an annotation is defined before the class itself. It seems like there is no error handling about this at all, even for the members. How to treat that?